Add relations of an individual (YAML specs v2.1.0)

NutthanichN opened the pull request:
Introduce YAML specs version 2.1.0

**Changes from v2.0.0**
- Add `relations` key in the `owl:Individual` section to keep the data about the individual and its relations

MaiMee1 commented on tests/specs/v2.1.0/test_case1.md:
```diff
@@ -0,0 +1,269 @@
+version v2.1.0
```
Readers of the markdown file could confuse this version with the ontology version.
```suggestion
mao specs version: v2.1.0
```

MaiMee1 commented on tests/specs/v2.1.0/test_case1.md:
```diff
@@ -0,0 +1,269 @@
+  - hasTopping
+  - hasBase
+
+## Data Property
+- TopDataProperty
+
```
The hierarchy doesn't show anything new, should we include this?

NutthanichN replied:
Since there's no data property specified in this test case, the hierarchy shouldn't be there either. I'll remove this section from .md file.

MaiMee1 commented on tests/specs/v2.1.0/test_case1.md:
```diff
@@ -0,0 +1,269 @@
+- TopDataProperty
+
+## Annotation Property
+- licence
```
"licence" is not the spelling Dublin Core use

MaiMee1 commented on tests/specs/v2.1.0/test_case1.md:
```diff
@@ -0,0 +1,269 @@
+  - hasTopping
+
+## PizzaBase
+### Annotations
+Comment
+  - Pizza dough that used as a pizza base
+### Description
```
There is no space after the comment annotations unlike others. Could this be confusing?

NutthanichN replied:
It could be. I'll add space to make it consistents with other sections.

MaiMee1 commented on tests/specs/v2.1.0/test_case1.md:
```diff
@@ -0,0 +1,269 @@
+
+# Object Properties
+## topObjectProperty
+## hasBase
+### Description
+Domain:
+  - Pizza
+
+Range:
+  - PizzaBase
+
+Sub-properties:
+  - topObjectProperty
+
```
hasBase doesn't have topObjectProperty as its sub-property? I think the section should be either "superproperties" or "subproperty of"

MaiMee1 reviewed (commented):
Currently both entities defined in the ontology and imported from elsewhere shows no prefix. I think this could be confusing. Maybe consider adding prefixes.

NutthanichN commented:
I edited .md test cases according to @MaiMee1 suggestions
- Add text to indicate that the version number is specs version
- Changed "Sub-properties" section to "Subproperty of"
- Added space after comment annotations

Please review again then I'll merge this branch to master.

th-bunratta reviewed (approved):
Looks good. Specs should usually be accompanied by a corresponding feature. Don't forget to resolve conflicts before merging.
